pkg/sdn/plugin/node_iptables.go

```diff
@@ -91,11 +91,10 @@ func (n *NodeIPTables) syncIPTableRules() error {
 // Get openshift iptables rules
 func (n *NodeIPTables) getStaticNodeIPTablesRules() []FirewallRule {
 	return []FirewallRule{
-		{"nat", "POSTROUTING", []string{"-s", n.clusterNetworkCIDR, "-j", "MASQUERADE"}},
-		{"filter", "INPUT", []string{"-p", "udp", "-m", "multiport", "--dports", VXLAN_PORT, "-m", "comment", "--comment", "001 vxlan incoming", "-j", "ACCEPT"}},
-		{"filter", "INPUT", []string{"-i", TUN, "-m", "comment", "--comment", "traffic from SDN", "-j", "ACCEPT"}},
-		{"filter", "INPUT", []string{"-i", "docker0", "-m", "comment", "--comment", "traffic from docker", "-j", "ACCEPT"}},
```

Comment:
Does docker actually create this rule?  I just started docker and I don't get that rule on my F25 box, but starting docker does create docker0.  Maybe we don't need it?  But we added it for a reason I think, though I don't recall what that reason was.